changelog/unreleased/bugfixes/6928.md

```diff
@@ -0,0 +1 @@
+- Fixed upcoming `RouteObject` incorrect location.
```

Comment:
This fix was alread cherry-picked to v2.10, wasn't it? In this case we can skip the changelog entry.

Reply:
seems so https://github.com/mapbox/mapbox-navigation-android/releases/tag/v2.10.1